docs/root/intro/arch_overview/security/rbac_filter.rst

```diff
@@ -0,0 +1,36 @@
+.. _arch_overview_rbac:
+
+Role Based Access Control
+=========================
+
+* :ref:`Network filter configuration <config_network_filters_rbac>`.
+* :ref:`HTTP filter configuration <config_http_filters_rbac>`.
+
+The Role Based Access Control (RBAC) filter checks if the incoming request is authorized or not.
+Unlike the external authorization, the check of RBAC filter happens in the Envoy process and is
+based on a list of policies from the filter config.
+
+The RBAC filter can be either configured as a :ref:`network filter <config_network_filters_rbac>`,
+or as a :ref:`HTTP filter <config_http_filters_rbac>` or both. If the request is deemed unauthorized
+by the network filter then the connection will be closed. If the request is deemed unauthorized at
+the HTTP filter the request will be denied with 403 (Forbidden) response.
+
+.. tip::
+  It is recommended that these filters are configured as the first filter in the filter chain so
+  that requests are authorized prior to rest of the filters processing the request.
+
+Policy
+------
+
+The RBAC filter checks the request based on a list of policies. A policy consists of a list of
```

Comment:
Can you potentially ref link to some of the relevant messages in this section and maybe elsewhere?

Reply:
Added link for `policies`, permissions`, `principals` and `shadow policy`.